src/Backend/Core/Engine/Authentication.php

```diff
@@ -313,16 +316,40 @@ public static function loginUser(string $login, string $password): bool
     {
         $db = BackendModel::get('database');
 
-        // fetch the encrypted password
-        $passwordEncrypted = static::getEncryptedPassword($login, $password);
+        // build fallback for migrating devs
+        // todo remove in next major release
+        $encryptedPassword = BackendUsersModel::getEncryptedPassword($login);
```

Comment:
In the original issue you said that this should be there. I don't understand what php 5.2 or 5.3 has to do with this?

This can just be removed in the next release, right?